pkg/k8shandler/common_test.go

```diff
@@ -353,6 +353,45 @@ func TestResourcesCommonResourceAndNodeLimitDefined(t *testing.T) {
 	}
 }
 
+func TestProxyContainerResourcesDefined(t *testing.T) {
+
+	imageName := "openshift/oauth-proxy:1.1"
+	clusterName := "elasticsearch"
+
+	expectedCPU, _ := resource.ParseQuantity("100m")
```

Comment:
we can use `MustParse` instead since we're giving it a static string value https://godoc.org/k8s.io/apimachinery/pkg/api/resource#MustParse